.github/workflows/ci.yml

```diff
@@ -491,10 +491,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
+      if: (!contains(github.event.head_commit.message, 'ci_filter') || contains(github.event.head_commit.message, matrix.name ))
```

Comment:
I tried to add this outside of the `steps` so that it would be required only once but the `matrix.name` does not seem to be available there.